phylogenetic/defaults/config.yaml

```diff
@@ -15,6 +15,7 @@ display_strain_field: "strain"
 filter:
   min_length: 500
   exclude: "defaults/exclude.txt"
+  query: "is_lab_host != True"
```

Comment:
Just noting the minor confusion I had when I was review this change.

The value is `true` within the produced metadata.tsv, but this filter still works because Augur's `read_metadata` (pandas) infers dtype of this column as booleans.

Reply:
Good to know! I was wondering why I couldn't get `query: "is_lab_host != 'true'"` to work.

Reply:
> I was wondering why I couldn't get query: "is_lab_host != 'true'" to work.

I guess I shouldn't say "still works" but that it _has_ to be formatted like this! 

If you want `is_lab_host != 'true'` to work, then you can add `--query-columns is_lab_host:str` to force the column dtype to be string.

Reply:
Got it working in https://github.com/nextstrain/lassa/pull/25/commits/b6669dd0127c327d9b2656462059f090d6bc1525

Not for this PR, but this does make me wonder if we should start switching over to the ["generalized subsampling logic" in WNV](https://github.com/NW-PaGe/WNV-nextstrain/pull/1/files), where all the filtering/subsampling params are in one line and can be combined together for contextualized subsampling.